tests/integration-tests/tests/create/test_create.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+
+import pytest
+from remote_command_executor import RemoteCommandExecutor
+
+from tests.common.assertions import assert_errors_in_logs
+from tests.common.utils import retrieve_latest_ami
+
+
+@pytest.mark.dimensions("eu-central-1", "c5.xlarge", "ubuntu1804", "slurm")
```

Comment:
Do we need to pass the instance type and scheduler in dimensions, but we don't use it in function and config file?

Reply:
If we want to pin the dimension to `@pytest.mark.dimensions("eu-central-1", "c5.xlarge", "ubuntu1804", "slurm")`, passing in the parameters is the only way I know to let `test_runner` collect the test.
If we don't pass `scheduler` and `instance`, the dimension has to be changed to `@pytest.mark.dimensions("eu-central-1", "*", "ubuntu1804", "*")`